src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/managehearings/ManageHearing.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.managehearings;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateSerializer;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.Court;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.DocumentCollectionItem;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.DynamicMultiSelectList;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.HasCaseDocument;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.YesOrNo;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class ManageHearing implements HasCaseDocument {
+
+    @JsonSerialize(using = LocalDateSerializer.class)
```

Comment:
Do you really need "manageHearing" prepended to all these properties? It seems redundant.